docs/multi-domains.md

```diff
@@ -0,0 +1,37 @@
+# Multi-domains
```

Comment:
Here we also need to mention the following:
- ILC detects domain from the request by using `Host` HTTP header
- Every registered domain has own set of routes. They do not overlap.
- If request is going to the domain which is absent in **Router domains** - routes w/o assigned domain will be used for routing.

Reply:
done